macro-fullcalendar-api/src/main/java/org/xwiki/fullcalendar/script/FullCalendarScriptService.java

```diff
@@ -70,4 +70,19 @@ public String getICalEvents(String iCalStringURL, Date startDate, Date endDate)
     {
         return fullCalendarManager.getICalEvents(new URL(iCalStringURL), startDate, endDate);
     }
+    /**
+     * Get the events from an iCal in a specified date interval. This method expands the recurring events.
+     *
+     * @param iCalStringURL the String representation of an iCal URL.
+     * @param startDate the start of the interval of the returned calendar events.
+     * @param endDate the end of the interval.
+     * @param collapse expands the recurring events if {@code true}, or collapses them into the parent otherwise.
```

Comment:
Isnt it the other way around? If collapse is false, then the recurring events are expanded.